### ISSUE TITLE
Deeply nested package structure

### ISSUE THREAD

**Issue by MrPowers** [MrPowers/spark-daria#101]
spark-daria follows the standard Scala / Java deep nesting package convention that's annoying when importing code.

Users currently need to import code like this: `import com.github.mrpowers.spark.daria.sql.ColumnExt._`

I noticed that some libraries are deviating from these Scala conventions and offering imports like this: `import utest._`.

Maybe we can change the package structure so users can import code like `import mrpowers.daria.sql.ColumnExt._`?  Thoughts @nvander1 / @manuzhang...?

**Comment by manuzhang** [comment]
Yes, the package name is too long although I don't have a good idea for a shorter one. Maybe `daria._` is enough ?
